pandas/tests/dtypes/test_common.py

```diff
@@ -392,6 +392,48 @@ def test_is_not_unsigned_integer_dtype(dtype):
     assert not com.is_unsigned_integer_dtype(dtype)
 
 
+@pytest.mark.parametrize(
+    "dtype,expected",
+    [
+        ("int_", "int_"),
+        ("int8", "int8"),
+        ("Int8", "Int8"),
+        ("int16", "int16"),
+        ("Int16", "Int16"),
+        ("int32", "int32"),
+        ("Int32", "Int32"),
+        ("int64", "int64"),
+        ("Int64", "Int64"),
+        ("uint8", "uint8"),
+        ("uint16", "uint16"),
+        ("uint32", "uint32"),
+        ("uint64", "uint64"),
+        ("float16", "float16"),
+        ("float32", "float32"),
+        ("Float32", "Float32"),
+        ("float64", "float64"),
+        ("Float64", "Float64"),
+    ],
+)
+def test_all_type_comparison_with_true_result(dtype, expected):
```

Comment:
Please use only one variable here

Reply:
I didnt understand, can you explain what you mean?

Reply:
Dtype and expected is always the same. You can simply use dtype instead of both

Reply:
@pytest.mark.parametrize(
    "dtype",
    [
        ("int_"),
        ("int8"),
        ("Int8"),
        ("int16"),
        ("Int16"),
        ("int32"),
        ("Int32"),
        ("int64"),
        ("Int64"),
        ("uint8"),
        ("uint16"),
        ("uint32"),
        ("uint64"),
        ("float16"),
        ("float32"),
        ("Float32"),
        ("float64"),
        ("Float64"),
    ],
)
def test_all_type_comparison_with_true_result(dtype):
    assert pandas_dtype(dtype) == dtype


@pytest.mark.parametrize(
    "dtype,expected",
    [
        ("Int8", "int8"),
        ("Int16", "int16"),
        ("Int32", "int32"),
        ("Int64", "int64"),
        ("Float32", "float32"),
        ("Float64", "float64"),
    ],
)

Reply:
@phofl is it the change you want?

Reply:
Yes, but you can remove the brackets

Reply:
@pytest.mark.parametrize(
"dtype",
("int_"),
("int8"),
("Int8"),
("int16"),
("Int16"),
("int32"),
("Int32"),
("int64"),
("Int64"),
("uint8"),
("uint16"),
("uint32"),
("uint64"),
("float16"),
("float32"),
("Float32"),
("float64"),
("Float64"),
)
def test_all_type_comparison_with_true_result(dtype):
assert pandas_dtype(dtype) == dtype

@pytest.mark.parametrize(
"dtype,expected",
("Int8", "int8"),
("Int16", "int16"),
("Int32", "int32"),
("Int64", "int64"),
("Float32", "float32"),
("Float64", "float64"),
)

Reply:
like this?

Reply:
No, the round ones

Reply:
In the First test

Reply:
@pytest.mark.parametrize(
"dtype",
[
"int_",
"int8",
"Int8",
"int16",
"Int16",
"int32",
"Int32",
"int64",
"Int64",
"uint8",
"uint16",
"uint32",
"uint64",
"float16",
"float32",
"Float32",
"float64",
"Float64",
],
)
def test_all_type_comparison_with_true_result(dtype):
assert pandas_dtype(dtype) == dtype

@pytest.mark.parametrize(
"dtype,expected",
[
("Int8", "int8"),
("Int16", "int16"),
("Int32", "int32"),
("Int64", "int64"),
("Float32", "float32"),
("Float64", "float64"),
],
)

Reply:
is there any other problem?